keyboards/yd60mq/keymaps/via/keymap.c

```diff
@@ -0,0 +1,37 @@
+#include QMK_KEYBOARD_H
```

Comment:
Could you add a license header here?

Here's a sample one:

https://github.com/qmk/qmk_firmware/blob/0b2eae768637bc9230fec5d5167f617b21f4e179/quantum/template/base/keymaps/default/keymap.c#L1-L15

Reply:
Sure, just added a license in https://github.com/qmk/qmk_firmware/pull/10321/commits/7bd646ef1ff0ae18d508dc9c470d74c799d0f1f7